app/controllers/concerns/commentable_actions.rb

```diff
@@ -8,6 +8,7 @@ def index
 
     @resources = @resources.tagged_with(@tag_filter) if @tag_filter
     @resources = @resources.page(params[:page]).for_render.send("sort_by_#{@current_order}")
+    @resources = @resources.where(participatory_process_id: @participatory_process.id) if @participatory_process.present?
```

Comment:
Why do you scope to `participatory_process_id`? The id + type tuple will already be unique.